assets/xml/objects/object_um.xml

```diff
@@ -85,7 +94,19 @@
         <Texture Name="object_um_Tex_014CD0" OutName="tex_014CD0" Format="ci8" Width="32" Height="64" Offset="0x14CD0" />
         <Texture Name="object_um_Tex_0154D0" OutName="tex_0154D0" Format="ci8" Width="32" Height="64" Offset="0x154D0" />
         <Texture Name="object_um_Tex_015CD0" OutName="tex_015CD0" Format="rgba16" Width="32" Height="32" Offset="0x15CD0" />
-        <!-- <Blob Name="object_um_Blob_0164D0" Size="0x2000" Offset="0x164D0" /> -->
+
+        <Texture Name="gUmCremiaEyeOpenTex" OutName="cremia_eye_open" Format="ci8" Width="32" Height="32" Offset="0x164D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeHalfTex" OutName="cremia_eye_half" Format="ci8" Width="32" Height="32" Offset="0x168D0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyeClosedTex" OutName="cremia_eye_closed" Format="ci8" Width="32" Height="32" Offset="0x16CD0" TlutOffset="0x130D0" />
+        <Texture Name="gUmCremiaEyePleasedTex" OutName="cremia_eye_pleased" Format="ci8" Width="32" Height="32" Offset="0x170D0" TlutOffset="0x130D0" />
```

Comment:
Similarly, I just called this "Happy" instead of "Pleased". Which do you prefer? I can sync up ma1/ma2 with whichever you prefer, but we should all be consistent.